conf.yaml

```
@@ -794,8 +794,8 @@ contents:
             prefix:     en/ecctl
             tags:       CloudControl/Reference
             subject:    ECCTL
-            current:    v1.0.0-beta2
-            branches:   [ v1.0.0-beta2 ]
+            current:    v1.0.0-beta3
+            branches:   [ master, v1.0.0-beta3, v1.0.0-beta2 ]
```

```suggestion
            branches:   [ master, 1.0.0-beta3, 1.0.0-beta2, v1.0.0-beta3, v1.0.0-beta2 ]
```

(Temporary fix until we can add a redirect in to the docs in `1.0.0-beta3` and `1.0.0-beta2`.)